components/cam/bld/namelist_files/use_cases/2000_scream_hr_av1c-04p2.xml

```diff
@@ -70,39 +68,58 @@
 <!-- Macrophysics/microphysics coupling -->
 <cld_macmic_num_steps    > 6</cld_macmic_num_steps>
 
-<!-- External forcing for BAM or MAM -->
-<soag_ext_file	 ver="mam" >atm/cam/chem/trop_mozart_aero/emis/aces4bgc_nvsoa_soag_elev_2000_c160427.nc</soag_ext_file>
-
-<!-- Surface emissions for MAM4 -->
-<dms_emis_file	      >atm/cam/chem/trop_mozart_aero/emis/DMSflux.2000.1deg_latlon_conserv.POPmonthlyClimFromACES4BGC_c20160226.nc</dms_emis_file>
-
-<!-- Marine organic aerosol namelist settings -->
-<mam_mom_mixing_state>3</mam_mom_mixing_state>
-<mam_mom_cycle_yr  >1                                                                                    </mam_mom_cycle_yr  >
-<mam_mom_datapath  >'atm/cam/chem/trop_mam/marine_BGC/'                                                  </mam_mom_datapath  >
-<mam_mom_datatype  >'CYCLICAL'										 </mam_mom_datatype  >
-<mam_mom_filename  >'monthly_macromolecules_0.1deg_bilinear_latlon_year01_merge_date.nc'                 </mam_mom_filename  > <!-- Using the 2000 file, for now -->
-<mam_mom_fixed_tod >0											 </mam_mom_fixed_tod >
-<mam_mom_fixed_ymd >0											 </mam_mom_fixed_ymd >
-<mam_mom_specifier >'chla:CHL1','mpoly:TRUEPOLYC','mprot:TRUEPROTC','mlip:TRUELIPC'			 </mam_mom_specifier >
-
-<!-- Stratospheric ozone (Linoz) -->
-<chlorine_loading_file      >atm/cam/chem/trop_mozart/ub/EESC_1850-2100_c090603.nc</chlorine_loading_file>
-<chlorine_loading_fixed_ymd >20000101</chlorine_loading_fixed_ymd>
-<chlorine_loading_type      >FIXED</chlorine_loading_type>
-<linoz_data_cycle_yr        >2000</linoz_data_cycle_yr>
-<linoz_data_file            >linoz1800-2100_2006jpl_climo_1.9x2.5_26L_extended.c160204.nc</linoz_data_file>
-<linoz_data_path            >atm/cam/chem/trop_mozart/ub</linoz_data_path>
-<linoz_data_type            >CYCLICAL</linoz_data_type>
-
 <!-- sim_year used for CLM datasets and SSTs forcings -->
 <sim_year>2000</sim_year>
 
-<!-- Settings related to the NH dycore -->
+<!-- Settings related to the NH dycore at ne1024 -->
 <theta_advect_form>		1</theta_advect_form>
 <theta_hydrostatic_mode>	.false.</theta_hydrostatic_mode>
-<tstep_type>			7</tstep_type>
-<cubed_sphere_map>		0</cubed_sphere_map>
-<se_nsplit>			20</se_nsplit> 
+<tstep_type>			10</tstep_type>
```

Comment:
I think these dycore settings actually belong in `namelist_defaults_cam.xml` since they are specific to the grid, not to the compset.